$OpenBSD$

Fix warning: initialization from incompatible pointer type
Due to E_Gadcon_Client_Class change
Fix warning: unused parameter *

--- src/modules/tasks/e_mod_main.c.orig	Sun Nov 13 11:41:17 2011
+++ src/modules/tasks/e_mod_main.c	Mon Nov 14 12:08:00 2011
@@ -7,7 +7,7 @@
 static E_Gadcon_Client *_gc_init(E_Gadcon *gc, const char *name, const char *id, const char *style);
 static void _gc_shutdown(E_Gadcon_Client *gcc);
 static void _gc_orient(E_Gadcon_Client *gcc, E_Gadcon_Orient orient);
-static char *_gc_label(E_Gadcon_Client_Class *client_class);
+static const char *_gc_label(E_Gadcon_Client_Class *client_class);
 static Evas_Object *_gc_icon(E_Gadcon_Client_Class *client_class, Evas *evas);
 static const char *_gc_id_new(E_Gadcon_Client_Class *client_class);
 
@@ -299,7 +299,7 @@ _gc_orient(E_Gadcon_Client *gcc, E_Gadcon_Orient orien
    e_box_align_set(tasks->o_items, 0.5, 0.5);
 }
 
-static char *
+static const char *
 _gc_label(E_Gadcon_Client_Class *client_class __UNUSED__)
 {
    return _("Tasks");
